tests/data/npts_gdaswave.t00z.wcoast.0p16.f000.grib2.txt

```diff
@@ -0,0 +1,19 @@
+1:0:npts=36391
+2:15254:npts=36391
+3:37897:npts=36391
+4:53794:npts=36391
+5:69064:npts=36391
+6:79482:npts=36391
+7:91308:npts=36391
+8:108541:npts=36391
+9:116716:npts=36391
+10:128832:npts=36391
+11:140848:npts=36391
```

Comment:
If this is a data file that tests use to see if they came up with correct answers, please start the name of the file with "ref_".